Documentation/doc-github-rules.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Github basic process
+description: Describe basic GitHub rules 
+author: michael
+last update: 02/10/2022
+---
+
+# GitHub basic process
+
+**Table of contents**
+- [Create issues](#create-issues)
+- [Create a fork](#create-a-fork)
+  - [Create a pull request (PR) in the original repository from a fork](#create-a-pull-request-pr-in-the-original-repository-from-a-fork)
+- [Create pull request (PR) in the original repository](#create-pull-request-pr-in-the-original-repository)
+- [Tools](#tools)
+
+This document describes a short process to effectively use GitHub to achieve the following main goals:
+
+1. Proposing a change. 
+2. Reporting a bug. 
+3. Making impovement suggestions.   
+
+## Create issues
+
+You use the **Issues** mechanism to report problems, bugs and suggest general improvements. This is for both internal and external contributors. 
+
+This approach is useful if you want to 
+
+- Flag items that cannot be addressed by a single PR.
+- Request that may need to be analyzed to find the proper course of action.
+- Record information to put on backlog.  
+
+Please, include relevant details the rest of the team needs to understand in order to discuss the issue effectively. 
+
+For more information, see [Creating an issue](https://docs.GitHub.com/en/issues/tracking-your-work-with-issues/creating-an-issue). 
+
+
+## Create a fork
+
+You fork a repository to propose changes to the original repository.
+This is for external contributors that do not have the ability to create PRs directly against the original repository. It can also be used by internal contributors for tasks that cannot be addressed by issuing PRs against the original repository. 
+For more information, see [Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+This approach is useful if you want to 
+
+- Fix a pubished artifact.
+- Create a new artifact.
+- Improve overall readibility, usefulness, and quality.
+   
+In this case, it's good practice to regularly sync your fork with the original repository. 
+For more information, see [Syncing a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
+
+### Create a pull request (PR) in the original repository from a fork
+
+You can make any changes to a fork, including making branches and opening pull requests. 
+If you want to contribute to the original repository, you can send a request to the original author to pull your fork into their repository by submitting a pull request.
+
+For more information, see [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+If you use the Github desktop app, you can perform these steps:
```

Comment:
If you're going to show how to use the GitHub desktop app, it's only fair that you also explain how to use the standard Web Gui. Most Linux users use the web UI. IMO it's better to just provide Links to other sites which explain it rather than reproduce images here, it takes up space and these external screen grabs are "perishable" (apps can change over time, do you want to try to keep up with them?).

Reply:
See my comment in the fix.